lms/djangoapps/learner_home/mock_data.json

```diff
@@ -79,45 +82,48 @@
                 "isPassing": true
             },
             "entitlement": null,
-            "courseRun": {
-                "isStarted": true,
-                "isArchived": false,
-                "startDate": "11/11/1900",
-                "endDate": "11/11/4040",
-                "minPassingGrade": 70,
-                "homeUrl": "edx.com/courses/my-course-url/home",
-                "marketingUrl": "edx.com/courses/my-course-url/marketing",
-                "progressUrl": "edx.com/courses/my-course-url/progress",
-                "unenrollUrl": "edx.com/courses/my-course-url/unenroll",
-                "resumeUrl": "edx.com/courses/my-course-url/resume",
-                "courseId": "course-number1-course-id1"
-            },
-            "certificate": {
-                "availableDate": null,
-                "isRestricted": false,
-                "isEarned": false,
-                "isDownloadable": false,
-                "certPreviewUrl": "edx.com/courses/my-course-url/cert-preview"
-            },
             "enrollment": {
-                "accessExpirationDate": "11/11/3030",
-                "canUpgrade": true,
-                "hasStarted": false,
                 "coursewareAccess": {
+                    "isTooEarly": true,
                     "hasUnmetPrerequisites": false,
-                    "isTooEarly": false,
                     "isStaff": false
                 },
+                "accessExpirationDate": "3030-11-11T00:00:00Z",
+                "canUpgrade": true,
+                "hasFinished": false,
+                "hasStarted": false,
                 "isAudit": true,
                 "isAuditAccessExpired": false,
                 "isEmailEnabled": false,
                 "hasOptedOutOfEmail": false,
                 "isEnrolled": true,
                 "isVerified": false,
-                "lastEnrolled": null
+                "lastEnrolled": "2022-09-20T14:35:27.049Z"
+            },
+            "certificate": {
+                "availableDate": null,
+                "isRestricted": false,
+                "isEarned": false,
+                "isDownloadable": false,
+                "certPreviewUrl": "edx.com/courses/my-course-url/cert-preview"
+            },
+            "courseRun": {
+                "isStarted": false,
+                "isArchived": false,
+                "startDate": "3030-11-11T00:00:00Z",
+                "endDate": "4040-11-11T00:00:00Z",
+                "minPassingGrade": 0.7,
```

Comment:
A note that these are being passed as a string and should more accurately be of format `"0.70"`